### ISSUE TITLE
Migrate problem

### ISSUE THREAD

**Issue by FTCr** [gregmuellegger/django-sortedm2m#55]
>Before applying it, edit the migration file and change in the operations list migrations.AlterField to AlterSortedManyToManyField (import it from sortedm2m.operations). This operation will take care of changing the intermediate tables, add the ordering field and fill in default values.

After this and migrate.
```python
Applying shop.0032_auto_20150617_2132...Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Volumes/WDC/Documents/Development/Web/rifktk-virtualenv/lib/python3.4/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/Volumes/WDC/Documents/Development/Web/rifktk-virtualenv/lib/python3.4/site-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Volumes/WDC/Documents/Development/Web/rifktk-virtualenv/lib/python3.4/site-packages/django/core/management/base.py", line 390, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Volumes/WDC/Documents/Development/Web/rifktk-virtualenv/lib/python3.4/site-packages/django/core/management/base.py", line 441, in execute
    output = self.handle(*args, **options)
  File "/Volumes/WDC/Documents/Development/Web/rifktk-virtualenv/lib/python3.4/site-packages/django/core/management/commands/migrate.py", line 221, in handle
    executor.migrate(targets, plan, fake=fake, fake_initial=fake_initial)
  File "/Volumes/WDC/Documents/Development/Web/rifktk-virtualenv/lib/python3.4/site-packages/django/db/migrations/executor.py", line 110, in migrate
    self.apply_migration(states[migration], migration, fake=fake, fake_initial=fake_initial)
  File "/Volumes/WDC/Documents/Development/Web/rifktk-virtualenv/lib/python3.4/site-packages/django/db/migrations/executor.py", line 147, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/Volumes/WDC/Documents/Development/Web/rifktk-virtualenv/lib/python3.4/site-packages/django/db/migrations/migration.py", line 115, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/Volumes/WDC/Documents/Development/Web/rifktk-virtualenv/lib/python3.4/site-packages/sortedm2m/operations.py", line 12, in database_forwards
    to_model = to_state.render().get_model(app_label, self.model_name)
AttributeError: 'ProjectState' object has no attribute 'render'
```
django 1.8, python3. thanks.

**Comment by gregmuellegger** [comment]
Hi, the problem there is Django 1.8 for the moment. Nobody worked so far on support for it. Would like to give it a shot and try to make it work?

**Comment by gregmuellegger** [comment]
Actually I just fixed that ;) and released 1.0: https://pypi.python.org/pypi/django-sortedm2m/ Have fun!

**Comment by FTCr** [comment]
Thank u very much.
